Tools/Scripts/libraries/webkitcorepy/webkitcorepy/tests/string_utils_unittest.py

```diff
@@ -61,7 +61,7 @@ def test_pluralize(self):
         self.assertEqual(string_utils.pluralize(2, 'mouse', 'mice'), '2 mice')
 
     def test_list(self):
-        self.assertEqual(string_utils.join(['integer', 'string', 'float']), 'integer, string and float')
+        self.assertEqual(string_utils.join(['integer', 'string', 'float']), 'integer, string, and float')
```

Comment:
GitHub won't let me comment on Line 71 >:( 
Could we update `test_split` as well so `self.assertEqual(string_utils.split('integer, string and float'), ['integer', 'string', 'float'])` --> `self.assertEqual(string_utils.split('integer, string, and float'), ['integer', 'string', 'float'])`
for consistency? (It doesn't break anything!)

Reply:
I want to +1 this comment, but add that we want to test both the Oxford and non-Oxford comma cases since `split` is basically _undoing_ `string_utils.join`.

Reply:
I'll make sure this hasn't regressed! @JonWBedard other than `string_utils.split` and `ValidateCommitMessage`, is there some other codepath that might be worth double checking?

Reply:
@JonWBedard @briannafan fixed in https://github.com/WebKit/WebKit/commit/c392603ae623989367f90209a384990779e8160a. One last look?!